torch/utils/tensorboard/summary.py

```diff
@@ -19,7 +19,8 @@
 from tensorboard.plugins.text.plugin_data_pb2 import TextPluginData
 from tensorboard.plugins.pr_curve.plugin_data_pb2 import PrCurvePluginData
 from tensorboard.plugins.custom_scalar import layout_pb2
-
+from tensorboard.plugins.hparams.plugin_data_pb2 import HParamsPluginData, SessionEndInfo, SessionStartInfo
```

Comment:
Let's only import this when hparam is requested. Similar to ` from tensorboard.plugins.mesh.plugin_data_pb2 import MeshPluginData` down below in `mesh`.